### ISSUE TITLE
More CombinedActions nodes

### ISSUE THREAD

**Issue by andydandy74** [BadMonkeysInc/Dynanimator#4]
Now that we have a template for combining multiple actions in one node, we should make more of those nodes....
Also: Test whether we can get rid of the third custom node ("DynanimatorNestedFunction XXX") which would make editing those components much easier.

**Comment by andydandy74** [comment]
Added the `Phases And Camera Position With Fixed Target` node in 68982f38067c544ac4aaea1a52811e5f961ab39a

**Comment by andydandy74** [comment]
After #22, this should be considerably less work since we don't need any more `Dynanimator FunctionHelper` nodes that combine several operations.

**Comment by andydandy74** [comment]
Added `Element Transparency And Camera Position With Fixed Target` (2bbb319888863f59191cd7d8cbe7a5fff68e1dbe) and `Solar Study And Camera Position With Fixed Target` (bb50dfb915990dc97d6ba51efe799cfd47a314e5)

**Comment by rabiadhikari** [comment]
It would be cool to be able to animate the Section box in the perspective to get cross sectional view. Having the section box animate, rotate while camera in motion could be a fantastic story telling process.

**Comment by andydandy74** [comment]
Great suggestion @rabiadhikari - why didn't I think of that? :-) 
I've created a separate issue for the section box animation (#26). Once that's done we can combine it with camera movement. Just don't expect anything to happen soon - we're all rather busy at the moment...
